docs/user-guide/recipes.rst

```diff
@@ -363,6 +363,12 @@ the tag we specify does not exist, it will be created. If the tag exists
 however, it will be updated as tag name is unique per repository and can point
 to only one manifest.
 
+.. note::
+
+    Pulp now supports manifest schema1 and schem2 versions. So when tagging a manifest,
```

Comment:
i changed this, so we hande that on server side, it does not make sense to force user to provide schema version if the provided the digest of the manifests( which is the unique identifier)